### ISSUE TITLE
gentoo + 045e:02ea Microsoft Corp. Xbox One Controller (wired usb)

### ISSUE THREAD

**Issue by hyperlocalheroes** [atar-axis/xpadneo#480]
as you can imagine, i'm struggling to get this to work.
installed xpadneo and modprobe hid-xpadneo, no success in having this working through usb cable.
what am i missing here?

**Comment by kakra** [comment]
xpadneo cannot work with USB unless your USB device would be a HID device. But Xbox controllers in USB mode use GIP instead of HID. The in-kernel xpad driver does support the GIP mode.

I'm planning on implementing GIP later by converting GIP und HID with a user-space daemon. That way, we can also work with the Xbox dongle. But currently there's no concrete plans on when to do that.

**Comment by darkicekeel** [comment]
so, only bluetooth?
i'm waiting for a "supposed compatible" dongle. i'll see if xpadneo will work with that, as soon as it arrives

**Comment by kakra** [comment]
> so, only bluetooth?

Yes, in full context, it means exactly that. But xpadneo doesn't depend on Bluetooth, it is a HID driver. Actually, it doesn't even see the Bluetooth connection.

> i'm waiting for a "supposed compatible" dongle. i'll see if xpadneo will work with that, as soon as it arrives

It should work. Until then, the stock-kernel xpad driver will support USB mode of this controller. It should just work via USB, no matter if xpadneo is installed. Both drivers do not conflict if used/enabled/installed at the same time because xpad does NOT handle HID.

**Comment by darkicekeel** [comment]
otherwise, just shutting down bluez.
i'll try asap and update here if succeeded

**Comment by kakra** [comment]
You can follow up here if you're observing that behavior of the Bluetooth device staying visible despite being disconnected: https://github.com/atar-axis/xpadneo/issues/475#issuecomment-2208490082
